src/test/java/com/google/javascript/clutz/partial/requireType.js

```diff
@@ -0,0 +1,8 @@
+goog.module('some.module');
+
+const AType = goog.requireType('some.other.module');
```

Comment:
nit: test case might be more readable if you'd have `AType` on the right hand side in the module name as well.

Reply:
done